build.sbt

```diff
@@ -28,7 +28,7 @@ lazy val common = project
   .settings(moduleName := "frees-rpc-common")
   .settings(scalacOptions := Seq("-deprecation", "-encoding", "UTF-8", "-feature", "-unchecked"))
 
-lazy val freesV = "0.4.6"
+lazy val freesV = "0.4.7"
```

Comment:
I guess the new version of Freestyle core will be 0.5.0, due the breaking changes.